python/llm/example/GPU/PyTorch-Models/Model/llama2/README.md

```diff
@@ -67,3 +67,88 @@ What is AI?
 
 AI, or artificial intelligence, refers to the ability of machines to perform tasks that would typically require human intelligence, such as learning, problem-solving,
 ```
+
+## Example 2 - Low Memory Version: Predict Tokens using `generate()` API
+
+In the example [low_memory_generate.py](./low_memory_generate.py), we show a low memory use case for a Llama2 model to predict the next N tokens using `generate()` API, with BigDL-LLM INT4 optimizations on Intel GPUs.
```

Comment:
Sometimes your GPU does not have enough memory to load the entire 4-bit model in your GPU (e.g. 34B, 70B models). In this example, we show one way of how to load llama models with very low GPU memory consumption. However this could be much slower than the original way.  The implementation is adapted from xxx (the link).

Reply:
Done.